src/modules/universal-discovery/components/create/choose.language.component.js

```diff
@@ -55,7 +55,7 @@ export default class ChooseLanguageComponent extends Component {
             attrs.selected = true;
         }
 
-        return <option {...attrs}>{language.name}</option>;
+        return language.enabled ? <option {...attrs}>{language.name}</option> : null;
```

Comment:
Please make an early return, it's more readable.

Reply:
Changed in https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/252/commits/65e5e93880c0377e2d4ffef45d5abb77d8d1b0e7.